docs/repositories-configure/adjusting-quality-gates.md

```diff
@@ -1,6 +1,11 @@
 # Adjusting quality gates
 
-Adjust the **quality gates** to configure when Codacy reports your pull requests and commits as not up to standards.
+The **quality gates** of your repository configure when Codacy reports your pull requests and commits as not up to standards.
+
+When you add your repository to Codacy, it automatically follows the [default gate policy for your organization](../organizations/using-gate-policies.md). If you want to set different quality gates for the repository, [create a new organization gate policy](../organizations/using-gate-policies.md#creating) to apply to the repository.
+
+!!! important
+    Although you can define custom quality gate settings for specific repositories, we recommend that you always [use a gate policy](../organizations/using-gate-policies.md) defined by your organization to enforce consistent rules across all repositories.
```

Comment:
```suggestion
    Although you can define custom quality gate settings for specific repositories, we recommend that you always [use a gate policy](../organizations/using-gate-policies.md) defined by your organization to enforce consistent rules across multiple repositories.
```

Since we will allow multiple policies, it's not like all need to follow 1, but the ideia is that a group of repositories can follow the same rules

Reply:
Yes, nice catch! 🤓